molo/core/templates/patterns/basics/articles/article-page_variations/article-body.html

```diff
@@ -1,30 +1,38 @@
 {% load wagtailcore_tags wagtailimages_tags core_tags wagtailembeds_tags  %}
+{% is_via_freebasics  as is_via_freebasics %}
```

Comment:
@codiebeulaine why do we need this? we can simply use `is_via_freebasics ` in the template

Reply:
@saeedmb8 we need this here as we can't call the template tag output directly, we need to assign it to the variable